lib/tail-until.js

```diff
@@ -17,9 +17,12 @@ async function tailUntil (filePath, until, timeout = 5000) {
 
   return new B(async (resolve, reject) => {
     let started = proc.start(startDetector);
+
```

Comment:
I know you didn't write this particular section of code, but I'm pretty sure that `proc.start` includes the ability to send in a timeout, which would make the next few lines unnecessary...